packages/paste-website/src/pages/components/tooltip/changelog.mdx

```diff
@@ -0,0 +1,37 @@
+export const meta = {
+  title: 'Tooltip - Changelog',
+  package: '@twilio-paste/tooltip',
+  description: 'An accessible and styled tooltip component.',
```

Comment:
this wins the prize for least helpful description ever

Reply:
Swapped for the package description